docs/doxygen/config.doxyfile

```diff
@@ -2226,7 +2227,8 @@ SKIP_FUNCTION_MACROS   = YES
 # the path). If a tag file is not located in the directory in which doxygen is
 # run, you must also specify the path to the tagfile here.
 
-TAGFILES               = libraries/standard/coreMQTT/docs/doxygen/output/mqtt.tag=../../../../libraries/standard/coreMQTT/docs/doxygen/output/html
+TAGFILES               = libraries/standard/coreMQTT/docs/doxygen/output/mqtt.tag=../../../../libraries/standard/coreMQTT/docs/doxygen/output/html \
+                                   "libraries/aws/shadow/docs/doxygen/output/shadow.tag=../../../../libraries/aws/shadow/docs/doxygen/output/html"
```

Comment:
```suggestion
                         libraries/aws/device-shadow-for-aws-iot-embedded-sdk/docs/doxygen/output/shadow.tag=../../../../libraries/aws/device-shadow-for-aws-iot-embedded-sdk/docs/doxygen/output/html
```

Reply:
fix in [this commit](https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1226/commits/19dedddc029d17d1c5428d938981b1216acd1452).